Quaver.Shared/Screens/Options/OptionsDialog.cs

```diff
@@ -40,7 +40,8 @@ public override void CreateContent()
         /// <param name="gameTime"></param>
         public override void HandleInput(GameTime gameTime)
         {
-            if (KeyboardManager.IsUniqueKeyPress(Keys.Escape))
+            if (KeyboardManager.IsUniqueKeyPress(Keys.Escape) || MouseManager.IsUniqueClick(MouseButton.Left) &&
```

Comment:
This needs to handle the case of when dropdowns extend out of the menu (resolution/skin).

Reply:
Handling the `Clicked` event in the constructor should be enough which handle the depth of the buttons for you.

```cs
public OptionsDialog() : base(0)
{
    ...

    Clicked += (sender, args) =>
    {
        if (!Menu.IsHovered())
            Close();
    };
}
```